### ISSUE TITLE
Question: Display manufacture 

### ISSUE THREAD

**Issue by bladyle** [david4599/PricehaxBT#5]
Hi I have couple of this but I have remove the battery before I read that they have only RAM and now can't use your app.
Do find the manufacture of the display or some kind of datasheet so I can try to run it?

**Comment by david4599** [comment]
Hey, yeah. your tags are now bricked. Hopefully one day we can find a way to retrieve and flash the firmware again which is a difficult task since the controller architecture and the serial protocol are unknown.

I guess the display you are talking about is the e-ink type and I have actually no experience with that.
However, I remembered a little Twitter thread about driving the display of a SmartTag HD150:
https://twitter.com/szczys/status/1363219226752868352

**Comment by bladyle** [comment]
Yes it's a e-ink display but with the different number from the one from your link. 
Mine has marked with HINK-EO42A33 on the flex cable that is soldered to the PCB ..
